control/setup_control.go

```diff
@@ -160,9 +160,7 @@ func (p *PIDLoop) TunePIDLoop(ctx context.Context, cancelFunc context.CancelFunc
 				errs = multierr.Combine(errs, err)
 			}
 
-			if err := p.ControlLoop.MonitorTuning(ctx); err != nil {
-				errs = multierr.Combine(errs, err)
-			}
+			p.ControlLoop.MonitorTuning(ctx)
```

Comment:
why do we not need to check these errors anymore? should the function not return any?

Reply:
it was always returning nil, I think in some initial implementation it could return actual errors but as it got changed I never removed the returned error from `MonitorTuning` so it's now unnecessary